README.md

```diff
@@ -168,26 +168,45 @@ cp solr_configs/conf/schema.xml solr_scripts/schema.xml
 
 The Dockerized versions of Solr on the server need your attention, too. Follow these steps.
 
-1. Deploy the schema changes on the staging server. 
-2. Shell into the server, go to the `lametro-staging` repo, and remove and rebuild the Solr container.
+1. Deploy the schema changes on the staging server.
+2. Shell into the server as the ubuntu user, go to the `lametro-staging` repo, and remove and rebuild the Solr container.
 
 ```
-docker rm lametro-staging-solr
-docker-compose up solr-staging
+cd lametro-staging
+
+# we need to stop the container before removing it!
+sudo docker stop lametro-staging-solr  
+sudo docker rm lametro-staging-solr
+
+# -d is the daemon flag
+sudo docker-compose up -d solr-staging
+```
+
+3. Logout as the ubuntu user, then log in as the datamade user.
+4. Rebuild the index for the staging server:
 ```
+workon lametro-staging
+python manage.py rebuild_index --batch-size=200
+```
+
+Did everything work as expected? Great - now onto the production site.
+
+Make sure your changes are deployed to the production server (i.e. you've cut a release with your changes).
 
-3. Rebuild the index for the staging server: `python manage.py rebuild_index`.
+1. Contact the folks at Metro and let them know the search functionality and data import will be down for a short period.
```

Comment:
So, I think we can remove this step and the following one...you saw how quickly we can rebuild docker and rebuild the solr index! Instead, I'd rewrite these steps and suggest that you run the docker and Haystack commands *between crons*. Does that make sense?